doc/javascript/application.js

```diff
@@ -11589,27 +11589,27 @@ if ( typeof define === "function" && define.amd && define.amd.jQuery ) {
       e.preventDefault();
       try {
         parent.frames.list.$('#search input').focus().select();
-      } catch (_error) {}
+      } catch (undefined) {}
```

Comment:
__[jshint]__
*  Expected an identifier and instead saw 'undefined'.

*  Expected ')' and instead saw 'undefined'.

*  Expected '{' and instead saw ')'.

*  Expected an identifier and instead saw ')'.

*  Expected an assignment or function call and instead saw an expression.

*  Missing semicolon.